src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -1465,6 +1484,98 @@ private static BundleEntryComponent explanationOfBenefit(BundleEntryComponent pe
     return newEntry(person, bundle,eob);
   }
 
+  /**
+   * Map the ChargeItem into a FHIR ChargeItem resource, and add it to the given Bundle.
+   *
+   * @param rand           Source of randomness to use when generating ids etc
+   * @param personEntry    The Entry for the Person
```

Comment:
Would you elaborate what's person? Is it Patient? Or a Bundle Entry?

Reply:
The best way to describe this is to go way back and explain how the whole software works.
1. A `Person` object is a full representation of a generated individual (or patient, in this case interchangeable) within the sythea state-machine workflow (meaning that this person has no standard or json or anything). `./run_synthea` calls a Generator object and gives it all input parameters  
2. `./run_synthea` -> `createPerson(seed, demographics)` -> `export(person, exportConfig)` 
3.  The exporter checks validity of the process, and if all good, calls `FhirR4.convertToFHIR(person)`. At this stage, the person will be transformed from the state-machine output to a .JSON format with FHIR standard
4. The `personEntry` is the mapping of the person to the `Patient` resource
5. At the `convertToFhir()` stage, a bundle is generated and aggregated with the resource; this means that within this function, resources get generated from the original person for every possible resource, from `condition` to `chargeItem`. Every resource entry that are to be included in the bundle are called `bundleEntry`
This means that a `chargeItem bundleEntry` is a creation of the chargeItem resource with Fhir standard that is proceeded to be added to the person's Bundle

Reply:
I'll create a GDoc adding all this information and possible discussions regarding the chargeItem, the synthea output and verilyIG